felix/cmd/calico-bpf/commands/conntrack.go

```diff
@@ -719,3 +753,35 @@ func (cmd *conntrackStatsCmd) Run(c *cobra.Command, _ []string) {
 		log.WithError(err).Fatal("Failed to iterate over conntrack entries")
 	}
 }
+
+func getCTMap(version int, ipv6 bool) maps.Map {
+	var ctMap maps.Map
+	switch version {
+	case 2:
+		ctMap = conntrack.MapV2()
+	default:
+		if ipv6 {
+			ctMap = conntrack.MapV6()
+		} else {
+			ctMap = conntrack.Map()
+		}
+	}
+	return ctMap
+}
+
+func setCTMapSize(version int, ipv6 bool) error {
```

Comment:
since it always has to be called just before `getCTMap`, would it make sense to include it in `getCTMap` ? Or just use this function to return (map, err) ?

Reply:
Good catch!

It would be better to include it in `getCTMap`. Will adjust as you suggested ;)